CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 ### Announcements
 * Starting 1.30 Kubernetes version and 1.27 LTS versions, beta apis will be disabled by default, when you upgrade to [them](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#before-you-begin). There will be an option provided to explicitly enable beta apis closer to the 1.30 release.
 * Istio service mesh addon revision asm-1-19 is no longer supported. If you are still using this revision on your cluster, please upgrade for continued support. More information about mesh upgrades and version support can be found [here](https://aka.ms/asm-aks-upgrade-docs).
+* 1.30 is the next LTS version after 1.27. Customers will have atleast 6 months of migration time available between 1.27 LTS to 1.30 LTS - before the 1.27 LTS goes End of Life.
```

Comment:
1.30 is just released and 1.27 EOL is in July of next year. Customers have a full year+. Is that second message necessary? Just link the docs where that should be clear?

Reply:
Also typo space missing in "at least"

Reply:
Sure on it.

Reply:
The second message is because 1.30 becoming LTS and necessary work to allow for upgrade from 1.27 to 1.30 is in pipeline. what we want to guarantee is atleast 6 months of migration .

Reply:
i added july 2025 and added the links to the doc.